Policies/github-terms-of-service.md

```diff
@@ -11,14 +11,16 @@ productVersions:
 englishOnly: true
 ---
 
-Thank you for using GitHub! We're happy you're here. Please read this Terms of Service agreement carefully before accessing or using GitHub. Because it is such an important contract between us and our users, we have tried to make it as clear as possible. For your convenience, we have presented these terms in a short non-binding summary followed by the full legal terms.
+Read this Terms of Service agreement carefully before accessing or using GitHub. For your convenience, there is a short non-binding summary before the actual agreement.
 
-### Summary
+<BR>
+  
+<details><summary open="false"><b>Summary</b></summary><BR>
 
 | Section | What can you find there? |
 | --- | --- |
-| [A. Definitions](#a-definitions) | Some basic terms, defined in a way that will help you understand this agreement. Refer back up to this section for clarification. |
-| [B. Account Terms](#b-account-terms) | These are the basic requirements of having an Account on GitHub.  |
+| [A. Definitions](#a-definitions) | Just contains definitions. Definitions are of basic terms. Refer back up to this section for clarification on the meanings of basic terms used in the agreement. |
+| [B. Account Terms](#b-account-terms) | These are mostly the basic requirements of having an Account on GitHub. Some other things such as things about the administrative control of Accounts, are also in this section.|
```

Comment:
Really? An adverb, _mostly_, in an attempt to reduce fluff and use plain English.

Reply:
Hello @philoserf,

Thanks for having a look at my suggestions.

For context about the change you've highlighted, know that:

- I've been making many, many changes to the terms.
- I intend to make more changes.
- Because of the amount of change I'm doing, I'm bound to make at least a few mistakes along the way.
- **These changes are simply suggestions or recommendations**.
- Whilst reducing 'fluff' and using plain English are worthwhile aims in this PR, they are by no means the only or perhaps even the main goals of this PR (this PR aims at making the agreement more user friendly by employing a variety of methods).

So given the above context, let's look at the highlighted change again. I can't find any specific comments I previously made about this change. However, it appears likely that the presence of subsection B1 ("Account Controls") influenced my decision in adding the word 'mostly'. The subsection doesn't appear to be about the basic requirements of having an account on GitHub even though it is in section B. Therefore, to say that the terms of section B **are** the basic requirements of having an Account on GitHub, appears to be inaccurate.